2.0/m_globalmessageflood.cpp

```diff
@@ -146,8 +146,8 @@ class ModuleGlobalMsgFlood : public Module
 		ServerInstance->Modules->AddService(mf);
 		ServerInstance->Modules->AddService(mf.ext);
 
-	        /* Enables Flood announcements for everyone with +s +F */
-        	ServerInstance->SNO->EnableSnomask('F',"FLOODANNOUNCE");
+		/* Enables Flood announcements for everyone with +s +f */
+		ServerInstance->SNO->EnableSnomask('f', "FLOOD");
```

Comment:
Would we not want +fF for local and remote? or are you looking to keep current functionality

Reply:
That's not how that works. You enable the local version and it does both.